core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -197,7 +200,13 @@ public BlockWorkerClientServiceHandler getWorkerServiceHandler() {
   public void start(WorkerNetAddress address) throws IOException {
     mAddress = address;
     try {
-      mWorkerId.set(mBlockMasterClient.getId(address));
+      RetryUtils.retry(() -> mWorkerId.set(mBlockMasterClient.getId(address)),
+          TimeBoundedExponentialRetry.builder()
+              .withMaxDuration(
+                  Duration.ofMillis(Configuration.getMs(PropertyKey.WORKER_MASTER_CONNECT_TIMEOUT)))
+              .withInitialSleep(Duration.ofMillis(100))
+              .withMaxSleep(Duration.ofSeconds(5))
```

Comment:
Since we don't make these configurable, does it make sense to hardcode the client side ones as well?